commcare_connect/opportunity/tests/test_forms.py

```
@@ -798,3 +800,53 @@ def test_flw_queryset_filtering(self, opportunity):
+        form = AddTaskTypeForm(
+            data={
+                "task_unit": deliver_units[0].id,
+                "name": "My Task",
+                "description": "A description",
+                "case_property": "some_property",
+            },
+            opportunity=opportunity,
+        )
+        assert form.is_valid(), form.errors
+
+    def test_missing_required_fields(self, opportunity, deliver_units):
```

Ruff (0.15.2)

[warning] Unused method argument: `deliver_units`

(ARG002)